english/candyland/eating/rest/rest.md

```diff
@@ -0,0 +1,5 @@
+You rest a while. What do you want to do now?
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.